src/Compilers/CSharp/Test/Semantic/Semantics/Utf8StringsLiteralsTests.cs

```diff
@@ -1829,7 +1953,8 @@ static void Main()
 ";
             var comp = CreateCompilation(source, targetFramework: TargetFramework.NetCoreApp, options: TestOptions.DebugExe);
 
-            CompileAndVerify(comp, expectedOutput: @"ReadOnlySpan").VerifyDiagnostics();
+            // PROTOTYPE(UTF8StringLiterals) : Add an entry in "docs/compilers/CSharp/Compiler Breaking Changes - DotNet 7.md"?
+            CompileAndVerify(comp, expectedOutput: @"array").VerifyDiagnostics();
```

Comment:
>@"array" [](http://example.com/codeflow?start=51&length=8)

Since this is a change in behavior, please include a corresponding compilation using `TestOptions.Regular10`. (Will it report "feature is in preview"?)